src/api/public/apidocs-new/paths/search_owner.yaml

```diff
@@ -73,16 +80,70 @@ get:
         application/xml; charset=utf-8:
           schema:
             $ref: '../components/schemas/owner_collection.yaml'
+          examples:
+            only package:
+              summary: Search by package name
+              value:
+                owner:
+                  rootproject: ''
+                  project: games
+                  package: ctris
+                  person:
+                    name: Iggy
```

Comment:
💭  I like examples that work with build.o.o more :)

Reply:
I finally used real project/packages names but not user names. I hope you don't mind. My reasons:

- This documentation is not only for build.o.o but for any other instances.
- I prefer people consciously try out the right data. So people don't try out without thinking.
- The real data we use for the examples can be removed at some point from build.o.o, which will force us to update the documentation. 
- For consistency, the rest of the endpoints use fake data.
- I'm not giving values (examples) to the different parameters on purpose. Because there isn't a good way to provide the correct combination of parameters that makes the search work if users simply click on Execute. Instead, I'm adding examples of 200 responses and, based on those examples, people will need to fill in the parameters fields to try out anyway.

If we finally decide that using real data from build.o.o is the right thing, we can adapt the whole documentation at once in a future iteration.